src/views/SingleEventView.vue

```diff
@@ -61,6 +61,10 @@
           <p class="subheader">What</p>
           {{ singleEvent.details.description }}
         </div>
+        <div class="info-block">
+          <p class="subheader">Price</p>
+          ${{ Number.parseFloat(singleEvent.price / 100).toFixed(2) }}
```

Comment:
Participating families should not see the price because they don't have to pay it. Let's just only render this div for general members and admins.